source/socket-message.js

```diff
@@ -227,11 +226,6 @@ Skylink.prototype.startRecording = function (callback) {
  *     console.info("Recording session has stopped. ID ->", success);
```

Comment:
line 214 callback.success.link should be removed
line 218 callbackSuccessWhenLink should be removed